python/cuml/cuml/tests/test_fil.py

```diff
@@ -107,12 +106,15 @@ def _build_and_save_xgboost(
         params["objective"] = "reg:squarederror"
         params["base_score"] = 0.0
 
+    xgboost_params = {} if xgboost_params is None else xgboost_params
```

Comment:
Why is this needed?

Reply:
We cannot use `{}` as a default value for parameters. See https://stackoverflow.com/a/26320938.